pkgs/by-name/sb/sby/package.nix

```
@@ -93,7 +97,7 @@ stdenv.mkDerivation rec {
   };
 
   meta = {
-    description = "SymbiYosys (sby) -- Front-end for Yosys-based formal verification flows";
+    description = "Front-end for Yosys-based formal verification flows";
```

```suggestion
    description = SymbiYosys, a front-end for Yosys-based formal verification flows";
```

I actually like including useful search terms, and bending the guideline is in this case is beneficial